aleph-client/src/lib.rs

```diff
@@ -405,3 +409,78 @@ pub fn get_block_hash<C: AnyConnection>(connection: &C, block_number: BlockNumbe
             panic!("Failed to obtain block hash for block {}.", block_number);
         })
 }
+
+pub fn get_current_block_number<C: AnyConnection>(connection: &C) -> u32 {
+    connection
+        .as_connection()
+        .get_header::<Header>(None)
+        .expect("Could not fetch header")
+        .expect("Block exists; qed")
+        .number
+}
+
+pub fn get_stakers_as_storage_keys<C: AnyConnection>(
+    connection: &C,
+    accounts: &[AccountId],
+    era: EraIndex,
+) -> BTreeSet<StorageKey> {
+    accounts
+        .iter()
+        .map(|acc| {
+            connection
+                .as_connection()
+                .metadata
+                .storage_double_map_key("Staking", "ErasStakers", era, acc)
+                .unwrap()
+        })
+        .collect()
+}
+
+#[derive(Decode)]
+pub struct EraValidators {
+    pub reserved: Vec<AccountId>,
+    pub non_reserved: Vec<AccountId>,
+}
+
+pub fn get_current_era_validators(connection: &SignedConnection) -> anyhow::Result<Vec<AccountId>> {
```

Comment:
this and the following functions return `Result` but they always return the `Ok` option. consider just returning without the wrapping it in the `Result`

Reply:
Good catch, fixed.